client/src/MergeAlerts.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Alert } from 'react-bootstrap'
+import { alertTable } from './constants'
+
+class Alerts extends React.Component {
+  render () {
+    return (
+      <div>
+        {displayAlert(this.props.alertType)}
+      </div>
+    )
+  }
+}
+
+function displayAlert (alertType) {
+  if (alertType === alertTable.MERGED) {
+    return (
+      <Alert variant='success'>
+      Account has been successfully merged.
+      </Alert>
+    )
+  } else if (alertType === alertTable.BAD_URL) {
+    return (
+      <Alert variant='danger'>
+      Invalid URL.
+      </Alert>
+    )
+  } else if (alertType === alertTable.NONEXISTENT_ACCOUNT) {
+    return (
+      <Alert variant='danger'>
+      This account does not exist.
+      </Alert>
+    )
+  } else if (alertType === alertTable.ALREADY_MERGED) {
+    return (
+      <Alert variant='danger'>
+      This account has already been merged. Please choose an unmerged account.
```

Comment:
Tbh, if you think about it, a user should never really be able to access a merged account since they are automatically redirected to the parent account.

Reply:
Well, they still may have pasted the wrong link? Unless I'm thinking about this the wrong way